command.go

```diff
@@ -640,6 +640,95 @@ func (cmd *FloatCmd) readReply(rd *proto.Reader) error {
 	cmd.val, cmd.err = rd.ReadFloatReply()
 	return cmd.err
 }
+//------------------------------------------------------------------------------
+type SlowLog struct {
+	Id       int64
+	ExecTime int64
+	Time     int64
```

Comment:
Can this be more Go friendly

```
Time time.Time
Duration time.Duration
```

Reply:
Hey, can you do me a favor, merge it  and then change it for me?
I deleted the original repository, and can't append the change here now.


I've already prepared the code here:

```go
type SlowLog struct {
	Id       int64
	Time     time.Time
	Duration time.Duration
	Args     []string
}

func slowLogParse(rd *proto.Reader, n int64) (interface{}, error) {
	var slowLog SlowLog
	id, err := readString2Int(rd)
	if err != nil {
		return nil, err
	}
	slowLog.Id = id

	t, err := readString2Int(rd)
	if err != nil {
		return nil, err
	}
	slowLog.Time = time.Unix(int64(t), 0)

	execTime, err := readString2Int(rd)
	if err != nil {
		return nil, err
	}
	slowLog.Duration = time.Millisecond * time.Duration(execTime)

	args, err := rd.ReadArrayReply(stringSliceParser)
	if err != nil {
		return nil, err
	}
	slowLog.Args = args.([]string)
	return &slowLog, nil
}

```